helm/helm-push.bzl

```diff
@@ -56,6 +64,7 @@ helm_push = rule(
       "repository_url": attr.string(mandatory = True),
       "repository_username": attr.string(mandatory = True),
       "repository_password": attr.string(mandatory = True),
+      "repo_type": attr.string(values = ["museum", "nexus"], default = "museum")
```

Comment:
@vmax this looks good. Could you be so kind to add some documentation about the new 'repo_type' attribute, please?

Reply:
Many thanks for your contribution to this repo, btw :-)

Reply:
@gutmox addressed in https://github.com/masmovil/bazel-rules/pull/25/commits/16f5d117923b30527f3fa61b7b589f6cc42a3f3a